resources/scripts/ttx_diff.py

```diff
@@ -154,14 +155,20 @@ def run_ttx(font_file: Path):
 
 
 # generate a simple text repr for gpos for this font, with retry
-def run_normalizer_gpos(normalizer_bin: Path, font_file: Path):
-    out_path = font_file.with_suffix(".markkern.txt")
+def run_normalizer(normalizer_bin: Path, font_file: Path, table: str):
+    if table == "gpos":
+        out_path = font_file.with_suffix(".markkern.txt")
+    elif table == "gdef":
+        out_path = font_file.with_suffix(".norm_gdef.txt")
```

Comment:
nit: I like the idea of the normalizer outputs having a consistent naming pattern. Why do we call gpos markkern and gdef norm_gdef instead of something consistent?

Reply:
I'd avoided this because I thought it would screw up the next crater run (since the old run would produce the first name, and the second would produce the new name, and they wouldn't know about each other) but this filename isn't actually the name that we use in crater so it should be fine, can change `markkern` to `norm_gpos`.